definitions.h

```diff
@@ -312,6 +314,14 @@ class VertexShader
         }
 };
 
+float crossProduct(Vertex v1, Vertex v2)
```

Comment:
I like how you wrote the crossProduct function in a separate location instead of writing the calculation in-line in the triangle rasterization function.